en/flight_controller/omnibus_f4_sd.md

```diff
@@ -135,27 +135,41 @@ Here is an example implementation. I used a Spektrum plug to get 3.3v from the D
 ![Omnibus F4 SD Pullup Implementation](../../assets/flight_controller/omnibus_f4_sd/pullup.jpg)
 
 
-## CRSF Crossfire Telemetry
-In addition to the [FrSky Telemetry](../peripherals/frsky_telemetry.md), the Omnibus also supports TBS CRSF Crossfire telemetry.
-It is used to send telemetry data from the flight controller (the vehicles's attitude, battery, flight mode and GPS data) to the RC transmitter (Taranis).
+## RC Telemetry
 
-Benefits are:
+The Omnibus supports telemetry to the RC Transmitter using [FrSky Telemetry](../peripherals/frsky_telemetry.md) or [CRSF Crossfire Telemetry](#crsf_telemetry).
+
+
+### CRSF Crossfire Telemetry {#crsf_telemetry}
+
+TBS CRSF Crossfire telemetry is used to send telemetry data from the flight controller (the vehicle's attitude, battery, flight mode and GPS data) to the RC transmitter (Taranis).
+
+Benefits over FrSky telemetry include:
 - Only a single UART is needed for RC and telemetry.
 - The CRSF protocol is optimized for low latency.
 - 150 Hz RC update rate.
 - The signals are uninverted and thus no (external) inverter logic is required.
 
-It is best to use the [TBS Crossfire Nano RX](http://team-blacksheep.com/products/prod:crossfire_nano_rx), since it is specifically designed for small Quads
-(the referenced link also contains the documentation).
+For Omnibus we recommend the [TBS Crossfire Nano RX](http://team-blacksheep.com/products/prod:crossfire_nano_rx), since it is specifically designed for small Quads.
+
+On the handheld controller (e.g. Taranis) you will also need a [Transmitter Module](http://team-blacksheep.com/shop/cat:rc_transmitters#product_listing).
+This can be plugged into the back of the RC controller.
```

Comment:
So we talk about taranis here, and plugging into the back of the controller. However if you go to the TX link it doesn't explain what modules it can plug into - ie it is not clear what it is compatible with in terms of RC controllers either in terms of protocol OR physical connectors.

Perhaps all RC controllers have a standardized physical format for modules? 

If not, we need to confirm the supported set of RC controllers, ideally via link to docs.

Reply:
> Perhaps all RC controllers have a standardized physical format for modules?

Yes the standard is called JR.

Reply:
Thanks. I should add that in the RC documentation - we just hand-wave over this by suggesting people buy a setup RC Transmitter/Receiver bundle.

Reply:
I.e. this is good to go then from my side.